main/signers/hot/HotSigner/worker.js

```diff
@@ -1,5 +1,5 @@
 const crypto = require('crypto')
-const ethSigUtil = require('eth-sig-util')
+const { signTypedData } = require('@metamask/eth-sig-util')
```

Comment:
the worker has access to sensitive data so we just need to make sure every time we change or add a dependency here. this looks OK to me but just tagging @floating for double review

Reply:
Yeah, ideally we would use `ethers` for this too if we can.  As discussed last week I think we get this in and then update again to switch to ethers if possible.